### DISCUSSION TITLE
Lista o contenido desplegable en CSS

### DISCUSSION THREAD

**Discussion by Unravelrs** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#441] [Q&A]
- Buenos días, primero que nada quiero agradecerles por subir sus repos, me ayudaron a orientarme en momentos donde me perdí, a algunos les robé unos iconillos,  ya estoy por terminar mi práctica, solo tengo una duda en la parte de la lista desplegable, no se como hacerlo, algún consejo? estaba pensando en desplegarlo con hover pero no se si sea lo adecuado. esto es lo que llevo.
Gracias

[Avance](https://i.ibb.co/bBPnD0P/Screenshot-2022-03-17-at-09-52-29-Vacunaci-n.png)

**Comment by marco-cortes** [comment]
Lo puedes hacer con el height del contenido, lo pones en 0 con overflow-y: hidden, y con JS puedes hacer que al darle click al boton el contenido cambie a height: auto y al  volver a darle click volver a ponerle height: 0 y yap ;b

**Reply by Unravelrs** [reply]
Gracias, al final hice algo parecido pero lo logré p:
